apisix/init.lua

```diff
@@ -110,6 +110,11 @@ function _M.http_init_worker()
     end
     require("apisix.balancer").init_worker()
     load_balancer = require("apisix.balancer")
+
+    if core.config == require("apisix.core.config_xds") then
```

Comment:
We can call it like:
https://github.com/apache/apisix/blob/9d450d7fe3169a77727df28696d083809b93977a/apisix/init.lua#L83

Reply:
updated